Server/routes.js

```diff
@@ -124,9 +124,11 @@ module.exports = function (app, io) {
           text += currentUser.incompleteRecipes.map(function (c) {
             return c.recipeUrl;
           }).join("\n"); // \n
-          text += "Cheers!";
-          // TODO: click here to unsubscribe.
+          text += "\nCheers!";
+          text += "\n\nUnsubscribe: ";
+          text += "http://localhost:3000/api/unsubscribe?id=" + currentUser._id;
```

Comment:
@youngfreezy Note that here you need to use your domain (probably coming from a configuration file or something like that).